pennylane_qrack/qrack_device.py

```diff
@@ -189,6 +189,8 @@ def __init__(self, wires=0, shots=None, **kwargs):
             self.isHostPointer = options['isHostPointer']
         if 'noise' in options:
             self.noise = options['noise']
+            if (self.noise != 0) and (shots is None):
```

Comment:
Sorry, I changed my mind on this one: extension of Qrack standards to Python would imply that no parentheses on necessary on the full conditional clause as a whole, only. Otherwise, unless there is truly the potential to have commuting operating (which these are not), _make **all** order-of-operations **fully explicit**,_ according to Qrack standards. There will _never_ be a mistake or a misinterpretation.